components/atoms/Label/Label.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { styled } from 'nativewind';
+import { Text as NativeText } from 'react-native';
+import classNames from 'classnames';
+import { View } from 'components/atoms/View';
+
+const StyledText = styled(NativeText);
```

Comment:
To jest niepotrzebne (jw.)

Reply:
tego użyłem bo tekst w native musi byc otoczony componentem Text z nativa, no ale patrz pierwsdzy komentarz